packages/iris-grid/src/sidebar/CustomColumnBuilder.jsx

```diff
@@ -164,7 +167,15 @@ class CustomColumnBuilder extends Component {
     if (!isCustomColumnApplying) {
       return;
     }
-    this.setState({ isCustomColumnApplying: false });
+    this.setState(
+      { isCustomColumnApplying: false, isSuccessShowing: true },
+      () => {
+        setTimeout(
```

Comment:
Need to track this timeout and clear it on componentWillUnmount. Right now if you close this panel while it still showing you get an error:
```
index.js:1 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
    in CustomColumnBuilder (created by IrisGrid)
    in div (created by Page)
    in div (created by Page)
    in Page (created by IrisGrid)
```

Might be easier to test by increasing the timeout.